simulator/src/main/java/com/hedera/block/simulator/generator/BlockStreamManager.java

```diff
@@ -25,6 +25,11 @@
 /** The block stream manager interface. */
 public interface BlockStreamManager {
 
+    /**
+     * Initialize the block stream manager and load blocks into memory.
+     */
+    void init();
```

Comment:
Maybe change this to a default interface method like: `default void init() {}` so you don't have to override with a no-op comment in `BlockAsFileLargeDataSets`

Reply:
Thanks, good point. fixed